hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/codec/BytesCodec.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.file.impl.codec;
+
+import com.hedera.pbj.runtime.Codec;
+import com.hedera.pbj.runtime.io.ReadableSequentialData;
+import com.hedera.pbj.runtime.io.WritableSequentialData;
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.io.IOException;
+
+public class BytesCodec implements Codec<Bytes> {
```

Comment:
Shouldn't we be using `Bytes.PROTOBUF` instead of implementing this here?

Reply:
I didn't find one and I checked what the method is expecting and created one

Reply:
Bytes isn't a generated record (it's part of PBJ runtime), and does not have a .PROTOBUF member.

Reply:
😢 ...ah got it, I didn't mean we had to switch away from using `File` as a wrapper for each "chunk" of content, but this works too. 👍